learn/getting_started/customizing_relevancy.md

```diff
@@ -0,0 +1,135 @@
+
+By default, all attributes are displayed in search results but the `displayedAttributes` array in the `settings` object allows you to change that.
+
+If you only want to display the `title`, `poster`, and `overview` for the `movies` index:
+
+<CodeSamples id= "getting_started_update_displayedAttributes_md" />
+
+You can read more about displayed attributes in our [dedicated guide](/reference/features/field_properties.md#displayed-fields).
+
+## Distinct attribute
+
+If you have many documents that are almost identical, you may wish to only return one of them per search. `distinctAttribute` is a tool in the `settings` object that allows you to do just that.
+
+When you set one of your attributes as the `distinctAttribute`, Meilisearch will not return more than one document with the same value associated to that attribute.
+
+Suppose you have an e-commerce dataset with an index on jackets. There are several identical items with minor variations in color.
+
+```json
+[
+  {
+    "id": 1,
+    "description": "Leather jacket",
+    "brand": "Lee jeans",
+    "color": "brown",
+    "product_id": "123456"
+  },
+  {
+    "id": 2,
+    "description": "Leather jacket",
+    "brand": "Lee jeans",
+    "color": "black",
+    "product_id": "123456"
+  },
+  {
+    "id": 3,
+    "description": "Leather jacket",
+    "brand": "Lee jeans",
+    "color": "blue",
+    "product_id": "123456"
+  }
+]
+```
+
+If you searched for `lee leather jacket` with the default settings, you would get all three documents. But if you set `product_id` as the `distinctAttribute`, Meilisearch will only return one of those jackets.
+
+You can read more about displayed attributes in our [dedicated guide](/reference/features/distinct.md).
+
+## Searchable attributes
+
+[Documents](/learn/core_concepts/documents.md) in Meilisearch are composed of multiple fields. By default, Meilisearch looks for matches in every field, but the `searchableAttributes` array in the `settings` object allows you to change that.
+
+For example, if you search the `movies` index for `2012`, Meilisearch searches for `2012` in every field: the `title`, `overview`, `release_year`, etc.. If you just want to search in the `title` field:
+
+<CodeSamples id= "getting_started_update_searchableAttributes_md" />
+
+Meilisearch will now only consider `title` during search and you will see fewer results.
+
+::: note
+
+Meilisearch will still highlight matches in other fields, but they won’t be used to compute results.
+
+:::
+
+### The order of `searchableAttributes`
+
+The order of the `searchableAttributes` array corresponds to the order of importance of the attributes.
+
+```json
+  [
+    "overview",
+    "title", 
+  ]
+```
+
+With the above order, matching words found in the `overview` field will have a higher impact on relevancy than the same words found in `title`. You can read more about this in our [relevancy guide](/learn/core_concepts/relevancy.md#attribute-ranking-order).
+
+## Stop words
```

Comment:
Did we ever reach an agreement on how to write stop words/stopwords/stop-words?

Reply:
Nope, that issue is still out there somewhere 🎈